styles/lichess/catppuccin.user.css

```diff
@@ -393,8 +448,8 @@
       background: @mantle;
       color: @text;
     }
-    .rclock.running .time {
-      background: #384722; /* putting it back to default so that the time on the clock is readable */
+    .rclock.running .time, :root:has(body[data-theme="system"]) .rclock.running .time {
```

Comment:
Why is this `:root:has(body[data-theme="system"])` needed?

Reply:
I forgot to remove it